app/src/main/res/layout/levels_screen.xml

```diff
@@ -33,14 +33,33 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-        <Button
-            android:id="@+id/level_play"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="@string/play_text"
+        <GridView
+            android:id="@+id/levels_grid_view"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_marginTop="@dimen/fab_margin"
+            android:cacheColorHint="#FFFFFF"
+            android:clipToPadding="false"
+            android:horizontalSpacing="@dimen/fab_margin"
+            android:numColumns="3"
+            android:paddingLeft="@dimen/fab_margin"
+            android:paddingRight="@dimen/fab_margin"
```

Comment:
do we have a style which contains most of these properties?

Reply:
Created one. Fixed